main.ts

```diff
@@ -1,15 +1,21 @@
 import { Base64 } from "https://deno.land/x/bb64@1.1.0/mod.ts";
+import { existsSync } from "https://deno.land/std@0.212.0/fs/exists.ts";
```

Comment:
not sure why you are adding this

Reply:
This is used to check if newDir exists or not (folder where renamed images are going to be stored). If it does not exist it is going to be created if it does it is going to be skipped.